src/environments.js

```diff
@@ -161,7 +162,8 @@ defineEnvironment("cases", {
             type: "align",
             align: "l",
             pregap: 0,
-            postgap: fontMetrics.metrics.quad,
+            // TODO(kevinb) get the current style
+            postgap: fontMetrics.metrics.getQuad(Style.TEXT),
```

Comment:
Sounds a little dicey.  I'll add a more detail comment in here, but I think we can leave this issue for a future diff.